README.md

```diff
@@ -79,6 +79,8 @@ _*Any artifact that don't match the selector will be preserved._
 
 It is possible to use a wildcard character to select all artifacts for filtering, i.e. `*:*`.
 
+It is possible to use a semicolon-separated list to specify more than one value for the selector.
```

Comment:
```suggestion
It is possible to use a comma-separated list to specify more than one value for the selector.
```

Reply:
It was confused, but this change was in the filters section and not the ordering.